### ISSUE TITLE
Missing Ask-Astro alerts

### ISSUE THREAD

**Issue by sunank200** [astronomer/ask-astro#248]
**Describe the bug**

We have observed that there is missing error alerts for this [prompt](https://apache-airflow.slack.com/archives/C03T0AVNA6A/p1704289611100239). We should debug and make sure all relevant errors are logged in the channel.

**Expected behavior**

We should debug and make sure all relevant errors are logged in the channel.

**Comment by pankajkoti** [comment]
@pankajkoti and I paired on this for a while. However, we could not find a concrete reason. We will need to explore this more.

**Comment by pankajkoti** [comment]
We have disabled the previous [ask-astro-error](https://console.cloud.google.com/monitoring/alerting/policies/9781854877488907565?project=ask-astro) policy and have created a new [Error count exceeds 0 in 1 min rolling window](https://console.cloud.google.com/monitoring/alerting/policies/4664289510344751999?project=ask-astro) policy.

We observed we're getting alerts as per this new policy even for the question that the previous policy did not trigger the alert.

Hence, I am closing this issue. In case, we observe any other miss, I believe we can open a new issue with the specific miss to investigate further.
